src/ai/backend/manager/api/session_template.py

```diff
@@ -85,6 +81,10 @@ async def create(request: web.Request, params: Any) -> web.Response:
                 }
             )
             result = await conn.execute(query)
+            resp = {
+                "id": template_id,
+                "user": user_uuid,
```

Comment:
When `UUID` object is used as `user_uuid`, Python's JSON library will fail to render this dict as JSON string. Let's add a fallback method.
```suggestion
                "user": user_uuid if isinstance(user_uuid, str) else user_uuid.hex,
```